app/src/components/content/VideoPlayer.vue

```diff
@@ -33,6 +38,32 @@ function autoHidePlayerControls() {
 function playerPlayEventHandler() {
     hasStarted.value = true;
     playerUserActiveEventHandler();
+
+    // Watch for changes in appLanguageAsRef
+    watch(
+        appLanguageAsRef,
+        (newLanguage) => {
+            const audioTracks = (player as any).audioTracks();
+
+            if (audioTracks && audioTracks.length > 0) {
+                for (let i = 0; i < audioTracks.length; i++) {
+                    const track = audioTracks[i];
+                    if (newLanguage) {
+                        // Check if the track's language matches the new language code
+                        if (track.language === newLanguage.languageCode) {
+                            // Enable the matched audio track
+                            track.enabled = true;
+                            console.log(`Enabled audio track: ${track.label} (${track.language})`);
+                        } else {
+                            // Disable other audio tracks
+                            track.enabled = false;
+                        }
+                    }
+                }
+            }
```

Comment:
I am not sure this is the best way to set the language. We have more than one audio track per language (different qualities), and in this way we are not sure which one is being selected.

I think it should be possible to change the player language (https://videojs.com/guides/options/#language)